lib/rucio/core/replica_sorter.py

```diff
@@ -204,6 +257,8 @@ def sort_replicas(dictreplica: "Dict", client_location: "Dict", selection: "Opti
     # all sorts must be stable to preserve the priority (the Python standard sorting functions always are stable)
     if selection == 'geoip':
         replicas = sort_geoip(dictreplica, client_location, ignore_error=True)
+    elif selection == 'custom':
```

Comment:
Maybe `custom_table`? `custom` alone seems a bit ambiguous.

Reply:
I've renamed it to `custom_table`.